src/ConstantsMixins.scss

```diff
@@ -1,10 +1,24 @@
-﻿
+﻿//Please read README section on Styling before editing or creating SCSS
+
+// This is your home for all shared themes and styles, please import it
+
+
+
+//Size Reference
+$Scale: 1vw;
+
 /* Site colours */
-$background-blue: rgba(36, 30, 78, 1);
-$accent-color-red: rgba(150, 2, 0, 1);
-$secondary-color-orange: rgba(206, 108, 71, 1);
-$foreground-color-yellow: rgba(244, 162, 97, 1);
-$light-yellow-font: rgba(234, 218, 162, 1);
+$background-color-dark: rgb(23, 3, 29);
+$background-color-light: rgb(182, 188, 244);
+$accent-color-mars-red: rgb(255, 0, 18);
+$secondary-color-mid-tone: rgb(255, 84, 9);
+$foreground-color-light: rgb(244, 82, 79);
+$light-font: rgb(243, 215, 212);
+
+/* Rover Theme Colors */
+$curiosityThemeColor: #a1fea1;
+$spiritThemeColor: #ffa6bc;
+$opportunityThemeColor: #ffeb6b;
```

Comment:
Could we update this to be a little more consistent on conventions?

In this section we have
- Capitalised
- lowercase-with-hyphens
- camelCase

and also
- RGB
- Hex

as usual, I don't really mind what we pick, but its a bit easier if everything is the same!